lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -104,6 +104,35 @@ function _dosomething_northstar_build_http_client() {
 /**
  * Send user profile updates to northstar.
  */
+// function dosomething_northstar_update_user($form_state) {
+//   $user = $form_state['user'];
+//   $id = $user->uid;
+
+//   $ns_user = dosomething_northstar_build_ns_user($user, $form_state);
+
+//   $northstar = new Northstar();
+//   $client = _dosomething_northstar_build_http_client();
+
+//   $northstar_user_info = dosomething_northstar_get_northstar_user($id);
+//   $northstar_user_info = json_decode($northstar_user_info->data, TRUE);
+//   $northstar_user_info = (object) $northstar_user_info['data'][0];
+//   $ns_id = $northstar_user_info->_id;
+
+//   $response = drupal_http_request($client['base_url'] . '/users/' . $ns_id, array(
+//     'headers' => $client['headers'],
+//     'method' => 'PUT',
+//     'data' => json_encode($ns_user),
+//     ));
+
+//   if ($response->code === '200' && module_exists('stathat')) {
+//     stathat_send_ez_count('drupal - Northstar - user updated - count', 1);
+//   }
+//   elseif ($response->code !== '200') {
+//     watchdog('dosomething_northstar', 'User not updated : ' . $response->code, NULL, WATCHDOG_ERROR);
```

Comment:
@chloealee Oh hm, if you don't have the password available, then I don't think the `POST` will work.

In which case you'll then need to send the update as a `PUT`. Thing is, we only support the `_id` param for `PUT` right now which it sounds like isn't available to you. We'd have to modify the endpoint so that you could use something like the drupal_id. Then we could give you something like...

`PUT /users/drupal_id/:drupal_id`
`PUT /users/_id/:mongo_id`

Does this make sense to do?